hardhat.config.zksync.ts

```diff
@@ -116,6 +117,13 @@ const config: HardhatUserConfig = {
     ],
     deployments: {},
   },
+  dependencyCompiler: {
+    paths: [
+      "@venusprotocol/governance-contracts/contracts/Governance/AccessControlledV8.sol",
```

Comment:
I think we don't need this

I think we need to add this configuration (`dependencyCompiler`) to the file `hardhat.config.ts` to be able to deploy Prime to the rest of the networks

Reply:
[edf3870](https://github.com/VenusProtocol/venus-protocol/pull/507/commits/edf387052fa9e1d68e05bfd317b24ea3adcc9dc1)